config/config.go

```diff
@@ -0,0 +1,24 @@
+package config
+
+import "github.com/ElrondNetwork/elrond-go/core"
+
+const configPath = "./config/config.toml"
```

Comment:
this could have been a flag with a default value

Reply:
same, this will be added when integrating cli flags